components/BookCard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
```

Comment:
Normally, we don't need to do this. This should work just fine.

```suggestion
import React from 'react';
```

Reply:
I had someone point this out to me once: https://github.com/facebook/react/pull/18102